.github/workflows/bump-version.yml

```diff
@@ -2,14 +2,9 @@ name: Bump Cloudbeat version
 
 on:
   workflow_dispatch:
-    inputs:
-      cloudbeat_version:
-        description: "New cloudbeat major.minor version (e.g. 8.13)"
-        required: true
```

Comment:
this is actually just confusing. we now take the current version in `version/version.go` and bump it to get the next one.